apps/frontend/src/pages/auth/auth.tsx

```diff
@@ -41,12 +41,11 @@ const Auth: React.FC = () => {
 		return null;
 	};
 
```

Comment:
You should not leave method handleSignInSubmit an empty. You have to configure sign in action in redux store and call it in handleSignInSubmit just as it shown in method handleSignUpSubmit.

Reply:
Yes, I forgot about it. Thanks...